packages/server/tests/acceptance.spec.ts

```diff
@@ -185,384 +229,223 @@ describe('RPC Server Integration Tests', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getUncleByBlockHashAndIndex', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getUncleByBlockHashAndIndex', []);
         expect(res.data.result).to.be.null;
     });
 
     it('should execute "eth_getUncleByBlockNumberAndIndex"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getUncleByBlockNumberAndIndex', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getUncleByBlockNumberAndIndex', []);
         expect(res.data.result).to.be.null;
     });
 
     it('should execute "eth_getUncleCountByBlockHash"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getUncleCountByBlockHash', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getUncleCountByBlockHash', []);
         expect(res.data.result).to.be.equal('0x0');
     });
 
     it('should execute "eth_getUncleCountByBlockNumber"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getUncleCountByBlockNumber', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getUncleCountByBlockNumber', []);
         expect(res.data.result).to.be.equal('0x0');
     });
 
     it('should execute "eth_getWork"', async function () {
-        callUnsupportedRelayMethod(this.relayClient, 'eth_getWork', []);
+        utils.callUnsupportedRelayMethod(this.relayClient, 'eth_getWork', []);
     });
 
     it('should execute "eth_hashrate"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_hashrate', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_hashrate', []);
         expect(res.data.result).to.be.equal('0x0');
     });
 
     it('should execute "eth_mining"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_mining', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_mining', []);
         expect(res.data.result).to.be.equal(false);
     });
 
     it('should execute "eth_submitWork"', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_submitWork', []);
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_submitWork', []);
         expect(res.data.result).to.be.equal(false);
     });
 
-    it('should execute eth_sendRawTransaction legacy', async function () {
-        const res = await callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + legacyTransactionHex]);
-        expect(res.data.result).to.be.equal('0x9ffbd69c44cf643ed8d1e756b505e545e3b5dd3a6b5ef9da1d8eca6679706594');
+    it('should execute "eth_sendRawTransaction" for legacy transactions', async function () {
```

Comment:
I think that it would be best to assert that actually the sender got his balanced reduced and the `to` address got its balance increased with `1`. This would verify that the `sendRawTransaction` was processed, it did not error and returned the manually computed eth hash